src/lightwingAPI/values.yaml

```diff
@@ -55,8 +55,16 @@ postgresPassword:
   name: postgres
   key: postgres-password
 
+clickhouse:
+  enabled: "false"
+  host: ""
+  port: 9000
+  database: "ccm"
+  username: "default"
+
 lwdSecrets:
   faktoryPassword: ""
+  clickhousePassword: ""
```

Comment:
Instead of having clickhousePassword which we would have to maintain again. We can use the value from our existing secret.
```
clickhouse:
  clickhousePassword:
    name: clickhouse
    key: admin-password
```